pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/runtime.tsx

```diff
@@ -17,7 +17,62 @@ export default function (props: GraphDashboardProps) {
     </LineGraph>,
 
     <LineGraph
-      title="Memory Usage"
+      title="CPU Time (per-node)"
+      sources={nodeSources}
+      tooltip={`CPU usage per node, in CPU seconds per second. E.g. if a node has four
```

Comment:
It seems like we should either sum the user and system time or specifically call out that this is just user time.  I think I prefer that we sum them, since system time is nontrivial.